gdctx/store.go

```diff
@@ -15,14 +19,25 @@ func RegisterStorePrefix(prefix string) {
 }
 
 // InitStore is to initialize the store
-func initStore() {
-	Store = store.New(Restart)
+func InitStore(initPrefix bool) {
+	Store = store.New()
+
+	if initPrefix == true {
```

Comment:
On why this wasn't caught, `make verify` uses gometalinter, which I've set to ignore warnings for everything including golint. I'll probably change it some time later.